source/code-generation/query-converter/convert-queries.txt

```diff
@@ -0,0 +1,161 @@
+.. _rm-convert-queries:
+
+===============
+Convert Queries
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can convert SQL queries to MongoDB syntax with query converter. 
+Query converter considers the mapping rules and schema 
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+   .. step:: Open the SQL query converter view
+
+      a. If it is your first time using the query converter in your 
+         project, click :guilabel:`Paste SQL Query`.
+         If your project already has converted SQL code, click the 
+         :guilabel:`+ ADD` button on the left-hand pane next to 
+         :guilabel:`Queries`.
+
+   .. step:: Enter your SQL query
+
+      a. Copy your SQL query to your clipboard.
+      #. Under the :guilabel:`SQL Query` header, paste your SQL query
+         into the text box.
+   
+   .. step:: Convert and test syntax
+
+      a. Click the :guilabel:`Convert` button. Wait for the AI to 
+         convert your code. The converted MongoDB syntax code displays 
+         in the :guilabel:`Converted MongoDB Query` text box.
+      #. Click the :icon-fa5:`copy` icon to copy the MongoDB syntax 
+         to your clipboard.
+      #. Test and validate the generated MongoDB syntax.
+
+      .. tip:: 
+
+         .. include:: /includes/fact-query-converter-filter.rst
+
+Examples
+--------
+
+The following table shows examples of SQL queries converted to MongoDB 
+syntax using query converter:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - SQL Syntax
+     - MongoDB Syntax
+
+   * - 
+       .. code-block:: sql
+          :copyable: false
+
+          SELECT * 
+          FROM CUSTOMERS
+     - 
+       .. code-block:: 
+          :copyable: false
+
+          async function query(db) {
```

Comment:
[question]

This looks like Node driver syntax? Do we need to specify that anywhere?

Reply:
@tomhollander Do you have an opinion here?

Reply:
Per our discussion adhoc conversions are only planned for node syntax with lightweight javascript, will mark this resolved.